### PR TITLE
Fix more empty space in grade list 

### PR DESCRIPTION
**Pull request by Ahmed-Naguib93**
refs: [MBL-17726](https://instructure.atlassian.net/jira/software/c/projects/MBL/boards/476?assignee=712020%3A158831a2-3c34-417e-b104-55ca208bcd5f&selectedIssue=MBL-17726)
affects:Student , Parent
release note: Implemented new design on grade screen

## Screenshots

https://github.com/user-attachments/assets/9dd55a47-928d-4898-879d-44228a002eb2

## Checklist

- [ ] Tested on phone
- [ ] Tested on tablet
- [ ] Tested in dark mode
- [ ] Tested in light mode

[MBL-17726]: https://instructure.atlassian.net/browse/MBL-17726?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Core/Core/Grades/Model/GradeListInteractor.swift [hunk 1]
```diff
@@ -251,46 +251,39 @@ public final class GradeListInteractorLive: GradeListInteractor {
 
         var assignmentSections: [GradeListData.AssignmentSections] = []
         var overdueAssignments = GradeListData.AssignmentSections(
-            id: UUID.string,
+            id: "overdueAssignments",
             title: String(localized: "Overdue Assignments", bundle: .core),
```
Comment (Ahmed-Naguib93) [inline Core/Core/Grades/Model/GradeListInteractor.swift:255]: I set a static ID to avoid recreating it when the toggle action changes.

**Comment by inst-danger** [comment]
<details><summary>Parent Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>One or more files are below the minimum test coverage 50%</td>
    </tr>
  </tbody>
</table>


#### Release Note: 
Implemented new design on grade screen

#### Affected Apps: Student, Parent

[MBL-17726](https://instructure.atlassian.net/browse/MBL-17726)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.11% | 91.11% | -0%
Core/Core/Planner/CalendarEvent/Model/API/PostCalendarEventRequest.swift | 0% | 0% | 0%
Core/Core/Planner/CalendarEvent/View/CustomFrequencyComponents/OccurrencesCountInputDialog.swift | 0% | 0% | 0%
Core/Core/Planner/CalendarEvent/Model/Helpers/RecurrenceRule+SelectionDescription.swift | 43.33% | 43.33% | 0%
Core/Core/Grades/View/CustomSwipeAction.swift | 0% | 0% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ac17bad8e061b71f63fdd5eccacf2826ac77c362
</p>

**Review by vargaat — APPROVED** [review]
QA+1

**Pull request merged**